src/aind_morphology_utils/swc.py

```diff
@@ -0,0 +1,175 @@
+import logging
+from typing import Dict
+
+import networkx as nx
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class NeuronGraph(nx.DiGraph):
+    """
+    A specialized DiGraph for representing neuron reconstructions.
+    """
+
+    def __init__(self):
+        super().__init__()
+
+    @staticmethod
+    def from_swc(swc_file_path: str) -> "NeuronGraph":
```

Comment:
You could probably also use a "classmethod" instead of a "staticmethod" and have the offset be one of the class attributes. I'm not sure how the nx.DiGraph class is instantiated though.